src/cascadia/TerminalApp/Pane.h

```diff
@@ -33,7 +33,7 @@ class Pane : public std::enable_shared_from_this<Pane>
         Horizontal = 2
     };
 
-    Pane(const GUID& profile, const winrt::Microsoft::Terminal::TerminalControl::TermControl& control, const bool lastFocused = false);
+    Pane(const GUID& profile, const winrt::Microsoft::Terminal::TerminalControl::TermControl& control, Pane* const rootPane, const bool lastFocused = false);

```

Comment:
```suggestion
    Pane(const GUID& profile, const winrt::Microsoft::Terminal::TerminalControl::TermControl& control, Pane* const rootPane = nullptr, const bool lastFocused = false);
```
Thoughts?

Reply:
So it's rather a special case when create a root pane that we pass here a null (because we can't pass pointer to itself in constructor). Otherwise we should always pass the valid pointer, so that's we I left it not optional.